cmake/modules/version.cmake

```diff
@@ -16,18 +16,15 @@
 #   KERNEL_VERSION_EXTENDED_STRING    "1.14.99-extraver+7"
 #   KERNEL_VERSION_TWEAK_STRING       "1.14.99+7"
 #
-#   KERNEL_VERSION_MAJOR      1
-#   KERNEL_VERSION_MINOR        14
-#   KERNEL_PATCHLEVEL              99
-#   KERNEL_VERSION_TWEAK              07
+#   KERNEL_VERSION_MAJOR     1
+#   KERNEL_VERSION_MINOR     14
+#   KERNEL_PATCHLEVEL        99
+#   KERNEL_VERSION_TWEAK     07
```

Comment:
should be 7 not 07, that would actually treat it as octal...

Reply:
also text here needs updating

Reply:
Replaced the entire block with the numbers from `version.h`, order and all.

Reply:
which is not what is should be...

Reply:
The text there is fine, as `${type}_VERSION_TWEAK` is the output variable from the cmake module, even if the generated C define has a different name format.